### DISCUSSION TITLE
Can't find package:swiper_angular

### DISCUSSION THREAD

**Discussion by GRDtechlab** [nolimits4web/swiper#5465] [Q&A]
Can't find package:swiper_angular . I am using angular version 13.2 and swiper version 8.0.0 .

When Importing `import { SwiperModule } from 'swiper/angular'` I got error saying can not find package:swiper_angular. 
I tried to add path inside tsconfig.json file like below

```ts
    // tsconfig.json

    compilerOptions :{
   "paths": {
       "swiper_angular": ["node_modules/swiper/angular"]
     }
   }
```
This is also not working. Whats wrong with this configuration ? Can you spot any missing part ? 

I follow official doc :  https://swiperjs.com/angular
Here is reproduction of error for the same inside this stackblitz link
https://stackblitz.com/edit/angular-ivy-gqpzhp?file=tsconfig.json

**Comment by jjamaral** [comment]
+1 here. Same error. Can somebody help please? @nolimits4web
